AGENT.md

```
@@ -0,0 +1,124 @@
+failed. When LACP is down, VFs are set to "disable" state to force failover;
+when LACP is up, VFs are set to "auto" state to enable traffic.
+
+## Go Version
+
+- Go version: 1.23.0
+- Toolchain: go1.24.1
+- Read `go.mod` before making changes to understand current dependencies
+
+## Build and Development Commands
+
+### Build
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)